components/Vote/text.js

```diff
@@ -44,22 +45,32 @@ const PMedium = (props) =>
     '& strong': {
       fontFamily: fontFamilies.sansSerifMedium,
       fontWeight: 'normal'
+    },
+    '& ul': {
+      paddingLeft: 20
     }
   })}>
     {props.children}
   </div>
 
-export const Body = ({ dangerousHTML }) =>
-  <div>
-    {
-      dangerousHTML.split('\n\n')
-        .map((c, i) =>
-          <PMedium key={i}>
-            <RawHtml dangerouslySetInnerHTML={{ __html: c }} />
-          </PMedium>
-        )
-    }
-  </div>
+export const Body = withInNativeApp(({ dangerousHTML, inNativeApp }) => {
+  const html = inNativeApp
+    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank" /, '')
```

Comment:
RE `target="_blank" ` replace:
Needs `g` flag too, why with an space at the end? `<a href="x" target="_blank">` is valid too

Reply:
```suggestion
    ? dangerousHTML.replace(/'/g, '"').replace(/target="_blank"/g, '')
```